independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,201 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "This is a connector to the Open Trivia Database to allows access to it's fun trivia questions",
+    "version": "1.0",
+     "contact": {
+      "name": "Kiveshan Naidoo, Rishay Sunder",
+      "email": "kiveshannaidoo9@gmail.com"

```

Comment:
@Kiveshan, please add a "url", you can see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json)